libs/libarchfpga/src/physical_types.h

```diff
@@ -1567,6 +1567,12 @@ enum e_Fc_type {
  *                   relation to the switches from the architecture file,    *
  *                   not the expanded list of switches that is built         *
  *                   at the end of build_rr_graph                            *
+ * @param arch_wire_switch_dec: Same as arch_wire_switch but used only for   *
+ *                   decremental tracks if it is specified in the            *
+ *                   architecture file.                                      *
```

Comment:
Suggest adding: "If -1, this value was not set in the architecture file and arch_wire_switch should be used for decrementing wire segments."